grails-app/conf/UrlMappings.groovy

```diff
@@ -52,6 +52,7 @@ class UrlMappings {
 		"/api/v1/streams/$id/publishers"(controller: "streamApi", action: "publishers")
 		"/api/v1/streams/$id/status"(controller: "streamApi", action: "status")
 		"/api/v1/streams/$resourceId/keys"(resources: "keyApi", excludes: ["create", "edit", "update"]) { resourceClass = Stream }
+		"/api/v1/streams/$streamId/keys/$keyId"(method: "PUT", controller: "keyApi", excludes: ["create", "edit", "save", "index"], action: "update") { resourceClass = Stream }
```

Comment:
`excludes` not needed here. It should only be needed when you use the `resources` argument.